release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -118,7 +118,12 @@ To install a CSI driver on a cluster running on vSphere, you must have the follo
 * vSphere version 6.7 Update 3 or greater
 * VMware ESXi version 6.7 Update 3 or greater
 
-These components are still supported, but are deprecated; support for them will be removed in a future version of {product-title}.
+Components with versions lower than those above are still supported, but are deprecated. Support for them will be removed in a future version of {product-title}.
+
+[NOTE]
+====
+If your cluster is deployed on vSphere, and the preceding components are lower than the version mentioned above, upgrading from {product-title} 4.9 to 4.10 on vSphere is supported, but no vSphere CSI driver will be installed. Bug fixes and other upgrades to 4.10 are still supported, however upgrading to 4.11 will be unavailable.
+====
```

Comment:
should it say "upgrades to 4.11" rather than "upgrading to 4.11" ...?

Reply:
@gnufied Yeah, probably should be [update instead of upgrade](https://docs.openshift.com/container-platform/4.9/updating/understanding-openshift-updates.html). 

Also, for 4.10 we use _supported_, but 4.11 we use _unavailable_. Is there a difference here between _unsupported_ and _unavailable_?

Reply:
Well - unavailable in this case means, upgrade to 4.11 simply won't be possible. unsupported generally means - you can upgrade but at your own risk.